lib/guardian.rb

```diff
@@ -550,6 +550,10 @@ def can_mention_here?
     @user.has_trust_level_or_staff?(SiteSetting.min_trust_level_for_here_mention)
   end
 
+  def is_me?(other)
```

Comment:
Moved this method as public since I find it quite useful when it’s needed to know if `guardian` is the current user.